CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Releasing
+=========
+
+Summary
+-------
+
+- Create a release branch in the main repository, not in a fork.
+- Prepare a commit for a release candidate, get an approving review, and tag the commit which will upload artifacts to PyPI.
+- Notify people and give time for feedback.
+- Address concerns raised via new issues for later handling or PRs against the release branch.
+- Prepare a commit for a final release, get an approving review, and tag the commit which will upload artifacts to PyPI.
```

Comment:
```suggestion
- Prepare a commit for a final release and push it to the release branch
- Get an approving review for the final release.
- Tag the commit using the final release version. This will trigger the upload of artifacts to PyPI.
```

Reply:
```suggestion
- Prepare a commit for a final release and push it to the release branch.
- Get an approving review for the final release.
- Tag the commit using the final release version. This will trigger the upload of artifacts to PyPI.
```